src/routes.js

```diff
@@ -15,14 +15,14 @@ import NotFound from './components/notFound';
 
 // Routes definition
 export default (
-  <Router path='/' component={App}>
+  <Route path='/' component={App}>
```

Comment:
Yes, it totally makes sense... I am going to change this in other my projects! Thanks! :D